### ISSUE TITLE
Support Gateway 2024.3

### ISSUE THREAD

**Issue by azatsarynnyy** [eclipse/che#23274]
### Is your task related to a problem? Please describe

The latest version of the [Gateway connector plugin](https://plugins.jetbrains.com/plugin/24234-openshift-dev-spaces/versions?noRedirect=true) does not support the latest build of JetBrains Platform 243.

<img width="386" alt="image" src="https://github.com/user-attachments/assets/4d04df10-4645-4b0a-a058-fd2af95ed650">

![image](https://github.com/user-attachments/assets/fc8eecc3-7b0a-44d8-8307-d2b81c8117a5)

### Describe the solution you'd like

Release a new version of the plugin that supports the latest version of the JetBrains Platform - 243.*

### Describe alternatives you've considered

Downgrade the installed Gateway application version to 2024.2.3

**Comment by ibuziuk** [comment]
@azatsarynnyy is it still actual

**Comment by azatsarynnyy** [comment]
> @azatsarynnyy is it still actual?

@ibuziuk yes, I'm working on making the plugin compatible with the latest Gateway release.
This time it depends on https://github.com/eclipse-che/che/issues/23102

**Comment by azatsarynnyy** [comment]
Done in https://github.com/redhat-developer/devspaces-gateway-plugin/pull/100
